sw/src/index.js

```diff
@@ -0,0 +1,6 @@
+export { setUpServiceWorker } from './service-worker/service-worker.js'
+export { OfflineInterface } from './offline-interface/offline-interface.js'
+export {
+    checkForUpdates,
+    checkForSWUpdateAndReload,
+} from './lib/registration.js'
```

Comment:
(minor) we should probably use the `exports` field in `package.json` to expose browser and service worker context code separately.  I.E. `import { setUp } from '@dhis2/sw/service-worker'` and `import { checkForUpdates, offlineInterface } from '@dhis2/sw/browser'` or something like that....?

Reply:
Yeah I like that 👍